utils/staking-miner/src/main.rs

```diff
@@ -319,10 +319,12 @@ struct MonitorConfig {
 #[derive(Debug, Clone, StructOpt)]
 struct EmergencySolutionConfig {
 	/// The block hash at which scraping happens. If none is provided, the latest head is used.
+	#[allow(dead_code)]
 	#[structopt(long)]
 	at: Option<Hash>,
 
 	/// The solver algorithm to use.
+	#[allow(dead_code)]
```

Comment:
if unused should be removed?

Reply:
Yes, but this would change public API, hence we can't remove it like that without risking breakage.

Reply:
Unless there is a decision before CI passes, let's make this a follow-up and avoid any further friction for people getting red CI tests and the goto fix `merge master` doesn't cut it :)

Reply:
ack, thanks for the ping @ordian.